requirements.txt

```diff
@@ -34,7 +34,7 @@ django-downloadview<=1.9
 django-polymorphic==2.0.3
 django-tastypie<=0.14.0
 django-invitations<=1.9.2
-geonode-oauth-toolkit>=1.1.2rc0
+geonode-oauth-toolkit==1.1.2rc0
```

Comment:
This should be 
`geonode-oauth-toolkit==1.1.3`

Reply:
1.1.3 causes the error described by @tomasjuanl above:

```
Change that by hand to AutoField instead of BigAutoField (i inspected all classes in that module). 
That worked, and now i have the following error: 

Traceback (most recent call last):
  File "/home/administrator/env/local/lib/python2.7/site-packages/django/utils/autoreload.py", line 229, in wrapper
    fn(*args, **kwargs)
  File "/home/administrator/env/local/lib/python2.7/site-packages/django/core/management/commands/runserver.py", line 116, in inner_run
    self.check_migrations()
  File "/home/administrator/env/local/lib/python2.7/site-packages/django/core/management/commands/runserver.py", line 168, in check_migrations
    executor = MigrationExecutor(connections[DEFAULT_DB_ALIAS])
  File "/home/administrator/env/local/lib/python2.7/site-packages/django/db/migrations/executor.py", line 19, in __init__
    self.loader = MigrationLoader(self.connection)
  File "/home/administrator/env/local/lib/python2.7/site-packages/django/db/migrations/loader.py", line 47, in __init__
    self.build_graph()
  File "/home/administrator/env/local/lib/python2.7/site-packages/django/db/migrations/loader.py", line 185, in build_graph
    self.load_disk()
  File "/home/administrator/env/local/lib/python2.7/site-packages/django/db/migrations/loader.py", line 103, in load_disk
    migration_module = import_module("%s.%s" % (module_name, migration_name))
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/administrator/env/local/lib/python2.7/site-packages/oauth2_provider/migrations/0006_auto_20171214_2232.py", line 11, in <module>
    class Migration(migrations.Migration):
  File "/home/administrator/env/local/lib/python2.7/site-packages/oauth2_provider/migrations/0006_auto_20171214_2232.py", line 21, in Migration
    field=models.OneToOneField(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, to=oauth2_settings.REFRESH_TOKEN_MODEL, related_name='refreshed_access_token'),
  File "/home/administrator/env/local/lib/python2.7/site-packages/oauth2_provider/settings.py", line 118, in __getattr__
    raise AttributeError("Invalid OAuth2Provider setting: '%s'" % attr)
AttributeError: Invalid OAuth2Provider setting: 'REFRESH_TOKEN_MODEL'
```

Reply:
Uhm, is this happening on master (2.10) too?

REFRESH_TOKEN_MODEL is defined here https://github.com/GeoNode/geonode-oauth-toolkit/blob/openid-connect/oauth2_provider/settings.py#L32

we need to understand what causes such error. It looks to me a possible conflict version with 'django-oauth-toolkit'

I need to investigate more on this.

Reply:
I just pinned a new version, 1.1.4 , removing BigAutoField too.

Could you please give it a try?

`pip install geonode-oauth-toolkit==1.1.4`

Reply:
sorry version 1.1.4.1

Reply:
@afabiani you haven't pushed `1.1.4` to PyPI yet, but using the latest commit on master works: 

`git+https://github.com/GeoNode/geonode-oauth-toolkit.git@1fe832d#egg=geonode-oauth-toolkit`

We can update the pull request once you release the new `geonode-oauth-toolkit` version

Reply:
For the time being I had to revert this, since it is breaking old builds. This issue will need more investigation unfortunately.
Nevertheless, I still wan't able to reproduce the error you reported.

Can you provide more details on the installation/deployment process?
Which OS you used and which procedures to try to reproduce it.

Thanks again.

Reply:
Hi @afabiani!

The error appears with tag 2.8.0+thefinal0. I used the quick installation guide with some modifications (the URL's for downloading geoserver are not good anymore, etc). It is better described in my original comment. This happens in Ubuntu 16.04. 
This is NOT a problem in master branch, nor branch 2.8.x.

Since tags should't be modified, I'll leave it like that. In any case, what i would do is make another working tag for the 2.8 version. The key is to freeze properly all the dependencies to avoid this conflicts in the future, with dependencies upgrades.

Reply:
Hi @afabiani!

The error appears with tag 2.8.0+thefinal0. I used the quick installation guide (Ubuntu for development) with some modifications (the URL's for downloading geoserver are not good anymore, etc). It is better described in my original comment. This happens in Ubuntu 16.04.
This is NOT a problem in master branch, nor branch 2.8.x.

Since tags should't be modified, I'll leave it like that. In any case, what i would do is make another working tag for the 2.8 version. The key is to freeze properly all the dependencies to avoid this conflicts in the future, with dependencies upgrades.

Reply:
This error used to happen on `master` when running the docker compose build. It seems to have been fixed in recent commits. Feel free to close this.